proto/proto.bzl

```diff
@@ -190,6 +190,10 @@ rust_proto_library = rule(
     _rust_proto_library_impl,
     attrs = {
         "deps": attr.label_list(
+            doc = """
+                List of proto_library dependencies that will be built.
+                One crate for each proto_library will be created with the corresponding stubs.
+            """,
```

Comment:
I know that this was not document so far, but maybe we should add comment to the rust_deps attribute too (it's the rust compile dependency for the rust_proto_library target, e.g. the proto library itself).